tests/test_spider.py

```diff
@@ -134,7 +134,33 @@ def test_auto_auth_response(mocker):
     assert urls_are_equal(request.url, expected_url)
     assert request.method == "GET"
     assert request.headers == {}
-    assert not request.callback
+
+
+def test_analyze_urls(mocker):
+    spider = EdxSpider(email=None, password=None)
+    fake_json = {
+        "root": "coursename",
+        "blocks": {
+            "course block": {
+                "display_name": "displayname",
+                "block_id": "course",
+                "student_view_url": "http://localhost:8003/xblock/block-coursename+course+type@course+block@course",
+                "lms_web_url": "http://localhost:8003/courses/course-coursename/jump_to/block-coursename+course+type@course+block@course",
```

Comment:
what will happen if the API starts providing another url in the `course_block`? Will that get scraped?